x/fswap/keeper/grpc_query_test.go

```diff
@@ -0,0 +1,121 @@
+package keeper_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/Finschia/finschia-sdk/baseapp"
+	"github.com/Finschia/finschia-sdk/simapp"
+	sdk "github.com/Finschia/finschia-sdk/types"
+	bank "github.com/Finschia/finschia-sdk/x/bank/types"
+	"github.com/Finschia/finschia-sdk/x/fswap/keeper"
+	"github.com/Finschia/finschia-sdk/x/fswap/types"
+)
+
+func TestFSwapQueryTestSuite(t *testing.T) {
+	suite.Run(t, &FSwapQueryTestSuite{})
+}
+
+type FSwapQueryTestSuite struct {
+	suite.Suite
+
+	app             *simapp.SimApp
+	ctx             sdk.Context
+	queryClient     types.QueryClient
+	keeper          keeper.Keeper
+	swap            types.Swap
+	toDenomMetadata bank.Metadata
+	fromDenom       string
+	toDenom         string
+}
```

Comment:
Why not use `KeeperTestSuite` in keeper_test, it looks like same. And other modules like bank module also use same test Class in keeper.

Reply:
Okay, I'll check. I just wanted test-suite for grpc_query.
Since I don't expect more testcase with different pre-conditions, I'll check keeper's test-suite to use.

Reply:
After I checked, I think I'll keep `FSwapQueryTestSuite` for grpc_query test.
Their test-cases need a little different pre-condition and It's okay to have their own test-suite.

Reply:
But it seems `KeeperTestSuite` and `FSwapQueryTestSuite` does the same things, isn't it?
Both of them set a dummy initial swap data.

Reply:
No, They are different.
Test suite for Query doesn't need creation of accounts with initial balance and the test suite only requires a registered `Swap` which `KeeperTestSuite` does not.